software/readout_hex.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+order:
+title: Reading out .hex files
+heading: Reading out .hex files
+description: A method to read out the .hex file on the micro:bit
+permalink: /software/readout_hex/
+ref: software
+lang: en
+assigned-to: markw
+review-with: jonnya
+---
+
+
+# Overview
+
+There may be occasions when you want to take a look at the programming that is running on the micro:bit or if you have misplaced the original program. This method reads out the .hex file on the micro:bit to a file in your working directory.
+
+# Pre-requisites
+
+In order to complete this task you will require some additional tools and libraries:
+
+  - Unix like OS
```

Comment:
We'll probably have to say "Linux based" here, as otherwise we are including things like FreeBSD (which I am not sure if PyOCD supports, it's not listed on their README).